src/js/stores/AppsStore.js

```diff
@@ -134,6 +134,7 @@ function processApp(app) {
   app.healthWeight = getAppHealthWeight(app.health);
 
   app.tasks = lazy(app.tasks).map(function (task) {
+    task.id =  task.id || task.taskId;
```

Comment:
Perfect. Still, did you get to the bottom of finding out why `task.taskId` sometimes is there instead of `task.id`?